pkg/server/router/dwn_test.go

```diff
@@ -1,13 +1,15 @@
 package router
 
 import (
+	"os"
```

Comment:
I need to make sure I'm on the same linter

Reply:
![Screen Shot 2022-09-28 at 2 44 36 PM](https://user-images.githubusercontent.com/7622243/192894250-cc583894-f4f7-439e-b84a-fc4634c11681.png)